### DISCUSSION TITLE
The path to "phospho_rep1.pin" renders an error

### DISCUSSION THREAD

**Discussion by issararab** [wfondrie/mokapot#46] [General]
Hello, can you please correct the link to the "[phospho_rep1.pin](https://github.com/wfondrie/mokapot/raw/master/tests/data/phoshpo_rep1.pin)"?

The "data" folder seems removed. Need to see a sample input file format, please.
Thank you.

**Comment by wfondrie** [comment]
Sorry about that! It is now located here:

https://github.com/wfondrie/mokapot/raw/master/data/phospho_rep1.pin

I'll make a not to update the links in the documentation.

**Comment by wfondrie** [comment]
This has been resolved in the latest update, #51
